gearmand/tests/test_integration_e2e.py

```diff
@@ -0,0 +1,37 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.gearmand import Gearman
+
+from . import common
+
+
+def assert_metrics(aggregator):
+    tags = ['first_tag', 'second_tag']
+    service_checks_tags = ['server:{}'.format(common.HOST), 'port:{}'.format(common.PORT)]
```

Comment:
Same here, renaming the variable would make more sense. You can have a single `tags` variable.

Reply:
Right, could be simpler. See my last commit :)